make/stratos/metrics/stop

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/make/include/defaults"
+source "${GIT_ROOT}/make/include/has_namespace"
+source "${GIT_ROOT}/make/include/has_release"
+
+if has_release "${STRATOS_METRICS_HELM_RELEASE}"; then
+  helm delete --purge "${STRATOS_METRICS_HELM_RELEASE}"
+fi
+
+if has_namespace "${STRATOS_METRICS_NAMESPACE}"; then
+  kubectl delete namespace "${STRATOS_METRICS_NAMESPACE}"
+fi
```

Comment:
Just realized… don't we need to wait until the namespace actually goes away (rather than in `Terminating`)? But I guess we don't wait elsewhere, so meh.